packages/wallet/src/redux/actions.ts

```diff
@@ -117,6 +122,11 @@ export interface ChallengeExpiredEvent {
 // Constructors
 // -------
 
+export const multipleWalletActions: ActionConstructor<MultipleWalletActions> = p => ({
```

Comment:
Even though you are mimicking neighboring code, I am going to comment anyways :). It would be great to use more descriptive parameter names than `p`.

Reply:
Agreed! This pattern is prolific, but I think we could clear it up in a separate PR. Something like `props` would probably be an improvement?